Added an option to hide numpad for phones with a physical numpad

ronniedroid opened the pull request:
I added an option in the settings to hide the numbers in the, it only keeps the *, + and  #, keys.

Please note that I kept the + button as it is necessary, and the * and # I kept for style.  

This is a useful option for people with phones that have a physical dialpad. 

Possible improvements that are above my skills:
- Hide the * + and # keys too, and make long pressing 0 on the physical dialpad input a "+".
- Make the Physical dial button work as expected (Dial the inputted number)

tibbi commented on app/src/main/res/layout/activity_settings.xml:
```diff
@@ -227,6 +227,22 @@
+                        style="@style/SettingsCheckboxStyle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="Hide dialpad numbers" />
```
oh cmon, move this into the string file of all languages (in english) and fetch it from there

ronniedroid replied:
will do

tibbi commented on app/src/main/res/layout/dialpad.xml:
```diff
@@ -18,6 +18,7 @@
         android:layout_marginStart="@dimen/activity_margin"
         android:background="?attr/selectableItemBackgroundBorderless"
         android:contentDescription="@string/one"
+        android:visibility="visible"
```
no need to set Visible everywhere, that is the default one

ronniedroid replied:
I see, I will remove it.

ronniedroid commented:
I removed the unnesecarry visible atteributes and moved the "Hide dialpad numbers" to the strings wills of all the language, (and translated the Arabic one).

tibbi commented:
is the keyboard usually hidden by default in other apps?

ronniedroid commented:
> is the keyboard usually hidden by default in other apps?

I don't understand what you mean

tibbi commented:
If you check Google Phone or some other Dialer apps with the device that has numpad, do their built in dialpads hide the numbers like you are suggesting in this app?

ronniedroid commented:
No, they don't, no one thinks of phones with keys. People with such phones are stuck with closed source OEM provided ones. Are you thinking of making this option on by default on such devices?

tibbi commented:
not sure, so far nobody requested it. There are just a couple users with such devices I guess

ronniedroid commented:
Not many, but Simple apps are the goto to most of them. Besides, an option in the settings won't hurt, right?

tibbi commented:
sure, Im considering it

tibbi commented:
it crashes at long pressing Plus with the new setting enabled

ronniedroid commented:
The  crashing on long pressing the Plus button was fixed.

tibbi commented:
should be good, thanks
